libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -109,6 +116,13 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
   @Input()
   hasItemSpacing: boolean;
 
+  @HostBinding('class')
+  get _cssClass() {
+    if (this._isStandAloneEnabled) {
+      return `stand-alone-spacing-${this.standAloneSpacing}`;
+    }
+  }
+
```

Comment:
I decided to remove this and conditionally bind some classes in `list.component.html`, so the classes are added to the elements that actually needs the margin. I think that makes it easier to reason about, instead of the spacing being controlled by a class on the list itself :)